script_library/deployment-actions-common.sh

```diff
@@ -12,6 +12,11 @@ function build_images(){
     run_ansible ${socok8s_absolute_dir}/playbooks/generic-build_images.yml -e "build_osh_images=yes"
 }
 
+function configure_ccp_deployer() {
+    echo "Configure CCP deployer node"
+    run_ansible ${socok8s_absolute_dir}/playbooks/openstack-configure_ccp_deployer.yml
```

Comment:
can you rename this file to "generic" instead?

Reply:
done